files/default/restore_backup.sh

```diff
@@ -1,22 +1,31 @@
 #!/usr/bin/env bash
 set -e
 
-MYSQL_CLIENT=/srv/hops/mysql-cluster/ndb/scripts/mysql-client.sh
-NDB_RESTORE=/srv/hops/mysql/bin/ndb_restore
-
-unset -v backup_path
-unset -v node_id
-unset -v backup_id
-unset -v mgm_connection
-unset -v ndb_restore_exclude_tables
-unset -v ndb_restore_op
+## Expected directory structure for the backup directory
+## {ROOT_BACKUP_DIRECTORY}
+## \-> sql
+##    \-> schemata.sql
+##    \-> users.sql
+## \-> BACKUP
+##    \-> BACKUP-{BACKUP_ID}
```

Comment:
There is one more subdirectory with a name something like
BACKUP-PART_1_OF_4

Reply:
@mronstro how do you get this directory? It certainly doesn't exist in my experiments

Reply:
As far as I can see this is the case when enabling Parallel Backup https://docs.oracle.com/cd/E17952_01/mysql-8.0-en/mysql-cluster-backup-parallel-data-nodes.html

Reply:
OK now I got it. The parallel backup is enabled by default but we run with one LDM thread, thus only one part
```
2023-02-09 11:33:38 [ndbd] INFO     -- Auto thread config uses:
 1 LDM threads,
 1 Query threads,
 0 tc threads,
 2 Recover threads,
 1 main threads,
 0 rep threads,
 1 recv threads,
 0 send threads

Reply:
But I think it will work anyway because I'm copying everything under `BACKUP-{BACKUP_ID}` and then in restore phase pointing `ndb_restore` to that directory which knows how to handle parallel backups.

I'll test it in any case. Good catch

Reply:
Yep, restore should work with just pointing to the top-level directory, so my comment was mostly a docs
comment and the rest you have figured out I noted.